api/src/main/java/org/openmrs/module/dhisreport/api/DHIS2ReportingException.java

```diff
@@ -6,7 +6,7 @@
  *  DHIS2 Reporting module is free software: you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
  *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
+ *  (at your option) any later version.
```

Comment:
Isn't this is the same line changed?

Reply:
Yes. I will fix it.

Reply:
I tried to fix it. But the two lines are identical.